src/work-allocation-2/containers/messages-container/task-assignment-person-not-authorised/task-assignment-person-not-authorised.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+    selector: 'exui-task-assignment-person-not-authorised',
+    templateUrl: './task-assignment-person-not-authorised.component.html'
+})
+export class TaskAssignmentPersonNotAuthorisedComponent {
+
+  public returnUrl = '/work/my-work/list';
```

Comment:
If returnUrl is not set does it goto mywork by default?

Reply:
Thanks @udaydenduluri33, yes I have set it to be like that to handle the worst case scenario. I will check that with Tommy.

Reply:
I have done the fix now. If the page is refreshed by the user, then default url is considered as the return url as the router navigation extras are lost.